shell.qml

```diff
@@ -216,53 +217,170 @@ ShellRoot {
       return "PROCESSLIST_OPEN_SUCCESS"
     }
 
-    function close() {
-      if (processListModalLoader.item)
-        processListModalLoader.item.hide()
+    Variants {
+        model: Quickshell.screens
 
-      return "PROCESSLIST_CLOSE_SUCCESS"
+        delegate: TopBar {
+            modelData: item
+        }
     }
 
-    function toggle() {
-      processListModalLoader.active = true
-      if (processListModalLoader.item)
-        processListModalLoader.item.toggle()
+    Variants {
+        model: Quickshell.screens
 
-      return "PROCESSLIST_TOGGLE_SUCCESS"
+        delegate: Dock {
+            modelData: item
+            contextMenu: dockContextMenu
+            windowsMenu: dockWindowsMenu
+        }
     }
 
-    target: "processlist"
-  }
+    CentcomPopout {
+        id: centcomPopout
+    }
 
-  Variants {
-    model: Quickshell.screens
+    DockContextMenu {
+        id: dockContextMenu
+    }
 
-    delegate: Toast {
-      modelData: item
+    DockWindowsMenu {
+        id: dockWindowsMenu
     }
-  }
 
-  Variants {
-    model: Quickshell.screens
+    NotificationCenterPopout {
+        id: notificationCenter
+    }
 
-    delegate: VolumePopup {
-      modelData: item
+    Variants {
+        model: Quickshell.screens
+
+        delegate: NotificationPopupManager {
+            modelData: item
+        }
     }
-  }
 
-  Variants {
-    model: Quickshell.screens
+    ControlCenterPopout {
+        id: controlCenterPopout
+
+        onPowerActionRequested: (action, title, message) => {
+            powerConfirmModal.powerConfirmAction = action;
+            powerConfirmModal.powerConfirmTitle = title;
+            powerConfirmModal.powerConfirmMessage = message;
+            powerConfirmModal.powerConfirmVisible = true;
+        }
+        onLockRequested: {
+            lock.activate();
+        }
+    }
 
-    delegate: MicMutePopup {
-      modelData: item
+    WifiPasswordModal {
+        id: wifiPasswordModal
     }
-  }
 
-  Variants {
-    model: Quickshell.screens
+    NetworkInfoModal {
+        id: networkInfoModal
+    }
 
-    delegate: BrightnessPopup {
-      modelData: item
+    BatteryPopout {
+        id: batteryPopout
+    }
+
+    PowerMenu {
+        id: powerMenu
+    }
+
+    PowerConfirmModal {
+        id: powerConfirmModal
+    }
+
+    LazyLoader {
+        id: processListModalLoader
+
+        active: false
+
+        ProcessListModal {
+            id: processListModal
+        }
+    }
+
+    PomodoroPopout {
+        id: pomodoroPopout
+    }
+
```

Comment:
Pretty much no changes to shell.qml should be here besides the addition of Pomodoro stuff